src/runtime/BigInt.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020-present Samsung Electronics Co., Ltd
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
+ *  USA
+ */
+
+#ifndef __EscargotBigInt__
+#define __EscargotBigInt__
+
+#include "runtime/PointerValue.h"
+
+namespace Escargot {
+
+class BigInt : public PointerValue {
+public:
+    BigInt(VMInstance* vmInstance, uint64_t num);
+
+    void* operator new(size_t size);
+    void* operator new[](size_t size) = delete;
+
+private:
+    BigInt(VMInstance* vmInstance);
+
+    size_t m_tag;
+    VMInstance* m_vmInstance;
```

Comment:
Should `BigInt` hold the VMInstance address?
For the current pr, VMInstance is used only for BigInt constructor.
Would it be used during the runtime later?

Reply:
We should hold VMInstance pointer because
bf_t holds bf_context_t pointer what is located in VMInstance
if VMInstace is destoryed, bf_context_t will be invalid.